app/Lib/BulkLookup.php

```diff
@@ -39,6 +39,8 @@ public static function retrieveByCallsignOrEmail(array $people) : array
                     'status' => $person->status,
                     'email' => $person->email,
                     'last_worked' => last(Timesheet::findYears($person->id, Timesheet::YEARS_WORKED)),
+                    'vintage' => $person->vintage > 0,
```

Comment:
FWIW, it doesn't look like we usually coerce to boolean in API responses.

Reply:
Generally, we don't but it also doesn't matter here. Javascript and EmberJS's templates can handle boolean truthiness in the form of numerically values.

Reply:
Done, in new PR (which is branched without a fork): https://github.com/burningmantech/ranger-clubhouse-api/pull/1294